packages/zeta-app-contracts/scripts/address.helpers.ts

```diff
@@ -0,0 +1,31 @@
+import { networks } from "@zetachain/networks";
+import { isProtocolNetworkName, ZetaProtocolNetwork } from "@zetachain/protocol-contracts";
+import { readFileSync, writeFileSync } from "fs";
+import { network } from "hardhat";
+import { join } from "path";
+
+import addresses from "../data/addresses.json";
+
+export const getAppAddress = (address: string, network: ZetaProtocolNetwork): string => {
+  return (addresses["ccm"] as any)[network][address];
```

Comment:
I think it would be cool to type this one.

Reply:
done